src/AppInstallerCLIE2ETests/TestData/Manifests/TestExeInstaller_InstallMSIX.yaml

```diff
@@ -0,0 +1,20 @@
+Id: AppInstallerTest.TestExeInstallerInstallsMSIX
+Name: EXE Installer that Installs MSIX - extra
+Version: 1.0.0.0
+Publisher: AppInstallerTest
+License: Test
+Installers:
+  - Arch: x86
+    Url: https://localhost:5001/TestKit/AppInstallerTestExeInstaller/AppInstallerTestExeInstaller.exe
+    Sha256: <EXEHASH>
+    InstallerType: exe
+    PackageFamilyName: 6c6338fe-41b7-46ca-8ba6-b5ad5312bb0e_8wekyb3d8bbwe

```

Comment:
Is this the intended way of doing the correlation? I thought that's what the AppsAndFeaturesEntries were for, so that the different InstallerType could be specified. In fact, this isn't the first time I've seen the issue where an exe installs MSIX packages - 
* https://github.com/microsoft/winget-cli/issues/2804
* https://github.com/Trenly/winget-pkgs/blob/9cf38f7be8b2f18cab84cefed19b00d957c5e9db/manifests/m/Microsoft/WindowsAppRuntime/1/2/1.2.2/Microsoft.WindowsAppRuntime.1.2.installer.yaml#L12-L15

If this is the intended way, that's great, I would only ask that it be documented somewhere that the PackageFamilyName may be used for correlation even though it is outside of the AppsAndFeaturesEntries, since based on the current manifest documentation, I would have thought that it wouldn't be used. And then, since the package behaves differently based on the OS Version, that there would be an installer node for each `MinimumOSVersion` that has a unique `AppsAndFeaturesEntries` to make the correlation unambiguous.

Reply:
`AppsAndFeaturesEntries` is for the Add/Remove Programs data in the Uninstall registry key.  It doesn't contain the package family name:
https://github.com/microsoft/winget-cli/blob/531d2f1677717c5c4d59f407011e6390f8589506/schemas/JSON/manifests/v1.5.0/manifest.singleton.1.5.0.json#L568-L600

Yes, the WinApp SDK / Runtime is what caused us to open that value up to installer types that were not definitely MSIX based.  Where would you suggest that it be documented that `Installer > PackageFamilyName` be used when an MSIX is installed?

The OS version targeting portion is relevant to the real manifest in question, not the test one.  I don't think it should be necessary if we can help it though.

Reply:
Ah, I see, that clarifies things. I think it can be documented at pkgs in the manifest docs; I can do that after this is merged.

Thank you for the explanation!